sycl/test/basic_tests/image.cpp

```diff
@@ -73,6 +73,32 @@ int main() {
     });
   }
 
+  // image wioth write accessor to it in kernel
+  {
+    int NX = 1024;
+    int NY = 1024;
```

Comment:
Please make size of the image smaller since it doesn't change idea of test.

Reply:
Done